### PR TITLE
Create FeatureComparison.md

### PR DESCRIPTION
**Pull request by lwcorp**
Based on https://github.com/sandboxie-plus/Sandboxie/issues/2317#issuecomment-1363982291

### CODE REVIEW

**Review by isaak654 — CHANGES_REQUESTED** [review]
What do you think about adding a new reference to the Feature Comparison page in the [Help Topics](https://github.com/sandboxie-plus/sandboxie-docs/blob/859f0276682d36e96f201bfd3a663d5cd73619a6/README.md#help-topics)?

Any suggestion would be appreciated.

**Review by isaak654 — CHANGES_REQUESTED** [review]
I think the empty cells below should contain N/A, Not Available or the minus sign: 

empty_cells

**Review by isaak654 — CHANGES_REQUESTED** [review]
I would suggest to add the links for both Classic and Plus licenses somewhere:
https://github.com/sandboxie-plus/Sandboxie/blob/master/LICENSE.Classic
https://github.com/sandboxie-plus/Sandboxie/blob/master/LICENSE.Plus

**Comment by lwcorp** [comment]
Done, but I wasn't aware only Classic was true open source. Of course this means Classic Supported can be used commercially.

**Comment by isaak654** [comment]
Apparently the maintainer provided its own license for Plus (aka Sandman UI) in order to discourage plagiarism attempts or such.

However, I reverted the change, as it did not look great visually. But I'm going to write a new sentence outside the table.

**Comment by isaak654** [comment]
Any review about the last commit would be appreciated.

**Comment by lwcorp** [comment]
I would rephrase:
1. `custom license` to `custom non free commercial usage license`
1. `GPL-3.0 license` to `GPL-3.0+ open source license` (plus sign because the license mentions "or later version")

As for what you write, sounds quite anti open source. Open source always invites plagiarism  but somehow the entire world agrees to use it. But does it really mean a Sandboxie Classic license allows commercial use since it's still open source?

**Review by isaak654 — APPROVED** [review]

**Comment by isaak654** [comment]
Thanks for the suggestions, so the sentence itself (technicalities aside) is fine.

In my opinion, custom licenses are a way of driving developers away from the project, so I fully understand your concerns. To answer your questions, I think a public debate would be necessary.
